Files/DataModels/SidebarPinnedModel.cs

```diff
@@ -265,7 +265,8 @@ public async Task AddItemToSidebarAsync(string path)
                     Font = InteractionViewModel.FontName,
                     Path = path,
                     Section = SectionType.Favorites,
-                    Icon = GlyphHelper.GetIconUri(path),
+                    Icon = new Windows.UI.Xaml.Media.Imaging.SvgImageSource(GlyphHelper.GetIconUri(path)),
```

Comment:
We are going to start using regular system thumbnails for the sidebar, does it make sense to make this change being that it might not be an SVG source anymore?

Reply:
I think we should keep it for now, we can always remove it after we switch to the system icons